src/ARMJIT_Memory.cpp

```diff
@@ -628,8 +640,12 @@ void Init()
     FastMem7Start = mmap(NULL, AddrSpaceSize, PROT_NONE, MAP_ANON | MAP_PRIVATE, -1, 0);
 
     MemoryBase = (u8*)mmap(NULL, MemoryTotalSize, PROT_NONE, MAP_ANON | MAP_PRIVATE, -1, 0);
+    #ifdef __APPLE__
+        MemoryFile = shm_open("melondsfastmem", O_RDWR|O_CREAT, 0600);
```

Comment:
I think you need to add something like the current pid to make shared mem object unique. Otherwise all melonDS processes will share this => chaos.

Reply:
Do I add that to the name?

Reply:
yes

Reply:
done.